Ginger/GingerCore/Actions/ActAgentManipulation.cs

```diff
@@ -66,15 +66,25 @@ public override void Execute()
                     {
                         this.Error = "Agent not running";
                     }
-                   
-                    RunOnBusinessFlow.CurrentActivity.CurrentAgent.Close();
+                    if (RunOnBusinessFlow.CurrentActivity.CurrentAgent.Status == Agent.eStatus.FailedToStart)
+                    {
+                        RunOnBusinessFlow.CurrentActivity.CurrentAgent.ResetAgentStatus(RunOnBusinessFlow.CurrentActivity.CurrentAgent.Status);
+                        this.ExInfo = "Agent is not running, status is updated.";
```

Comment:
Looks good. Just change the message to "Agent is not running, failed to start status is reset."